includes/class-amp-theme-support.php

```diff
@@ -364,6 +364,23 @@ public static function get_theme_support_args() {
 		return $support[0];
 	}
 
+	/**
+	 * Gets whether the parent or child theme supports Reader Mode.
+	 *
+	 * True if the theme does not call add_theme_support( 'amp' ) at all,
+	 * and it has an amp/ directory for templates.
+	 *
+	 * @return bool Whether the theme supports Reader Mode.
+	 */
+	public static function theme_supports_reader_mode() {
```

Comment:
The class is called `AMP_Theme_Support`, and this is part of the fully qualified name. So the method name here is:
```php
AMP_Theme_Support::theme_supports_reader_mode()
```
which seems unnecessarily verbose. The entire class is about what the theme supports, so that prefix does not provide any added value.

Some alternatives for more natural reading would be:
```php
AMP_Theme_Support::supports_reader_mode()
```
or even
```php
AMP_Theme_Support::for_reader_mode()
```

The end goal should always be to have the code read as naturally as possible (while still being clear and precise, of course).

Reply:
Great point. I'll change the name of this.

Reply:
https://github.com/ampproject/amp-wp/pull/3188/commits/2bec6cc7e84615d6a5d151f4b606302304fed9f8 changes the name to `AMP_Theme_Support::supports_reader_mode()`